app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -467,38 +467,50 @@ public void showAlert(Bitmap screenshot){
 
     @OnClick(R.id.images_upload_info)
     public void showUploadInfo(){
-        launchAlert(getResources().getString(R.string.images_uploaded)
-                ,getResources().getString(R.string.images_uploaded_explanation));
+        launchAlertWithLink(getResources().getString(R.string.images_uploaded)
+                ,getResources().getString(R.string.images_uploaded_explanation)
+                , "https://commons.wikimedia.org/wiki/Commons:Project_scope"
```

Comment:
Thanks @gerardolgvr , all looks good except hardcoded urls. Can you use final variables instead?

Reply:
Sure! I'm gonna do the changes. Thanks for the review